node-wit/index.d.ts

```diff
@@ -0,0 +1,81 @@
+// Type definitions for node-wit 4.2
+// Project: https://github.com/wit-ai/node-wit#readme
+// Definitions by: Julien Dufresne <https://github.com/julienduf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as Promise from "bluebird";
```

Comment:
The project itself doesn't use bluebird, and since this targets ES6, you should be able to just use `Promise`.